docs/list-available-operators.md

```diff
@@ -0,0 +1,29 @@
+# How do I list Operators available to install?
+
+There is an `APIService` in OLM named `PackageManifest` that contains information existing `CatalogSources`, which is essentially a repository of CSVs, CRDs, and packages that define an application, and their `ConfigMaps` in the cluster. By querying that API, you can see the list of available operators.
+
+## PackageManifest Commands
```

Comment:
I think we need a note here about global / namespace catalogs? And that packagemanifests lists the union of global + the namespace you're requesting, so you can see the list of operators to install in the requested namespace.

Reply:
Resolved.